test/util/testnode/full_node.go

```diff
@@ -141,7 +141,9 @@ func DefaultParams() *tmproto.ConsensusParams {
 
 func DefaultTendermintConfig() *config.Config {
 	tmCfg := config.DefaultConfig()
-	tmCfg.Consensus.TimeoutCommit = time.Millisecond * 300
+	// Reduce the target height duration so that blocks are produced faster
+	// during tests.
+	tmCfg.Consensus.TargetHeightDuration = time.Millisecond * 300
```

Comment:
I hope that's not going to be too fast. I guess if consensus can't handle it, it won't be that bad